.github/workflows/ci.yml

```diff
@@ -138,6 +74,13 @@ jobs:
       with:
         command: check
         args: --workspace --all-features --bins --tests --examples --benches
+    
+    - name: clippy
+      uses: actions-rs/cargo@v1
+      if: matrix.os == 'ubuntu-latest'
```

Comment:
Why not run clippy on all platforms?  We've already seen windows-only clippy errors, because of optionally compiled code around unix sockets.

Reply:
good point.